balancer/weightedroundrobin/balancer.go

```diff
@@ -420,6 +428,7 @@ type picker struct {
 	// The following fields are immutable.
 	target          string
 	locality        string
+	clusterName     string
```

Comment:
Does it make more sense to store these as a single `[]string` named `metricLabels` or something like that? I don't see these used for any other purpose.

The `Record` method on the metric descriptors accept a variadic list of labels as the last argument. So, if we have a string slice here to store all the labels, we could just pass it as `metricLabels...` to the `Record` method.